packages/elements/src/internals/machines/sign-in.machine.ts

```diff
@@ -146,90 +165,90 @@ export const SignInMachine = setup({
   context: ({ input }) => ({
     clerk: input.clerk,
     currentFactor: null,
-    environment: input.clerk.__unstable__environment,
```

Comment:
Just curious, why'd you remove the explicit `environment` entry in the context?

Reply:
Just trying to avoid maintaining multiple potential avenues for the same information. I plan on eventually pushing a PR around "graduating" `__unstable__environment` to just `environment` and giving it proper types as it's be "unstable" for three year.